### DISCUSSION TITLE
Folder directory

### DISCUSSION THREAD

**Discussion by zeeslikethat** [MissingCore/Music#44] [General]
I love this app, i use the universal version on my S21 Ultra til i can get hold of a Nothing phone (i live in Morocco) i just can't change directory to my music folder xD

**Comment by cyanChill** [comment]
@zeeslikethat `v1.0.0-rc.11` has been released and contains a new allowlist/blocklist feature that should let you select where music will be found (defaults to the `Music` directory on your device).
